### PR TITLE
Update Versions and Environment Vars

### PR DESCRIPTION
**Pull request by colin-msphere**
Updated:

- Jenkins to 2.32.3
- mesos plugin to 0.14.1
- ssh plugin to 1.14

Added an additional script that writes credentials to file instead of remaining in environment variables.

### CODE REVIEW

File: scripts/dcos-account.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#
+# Write DC/OS credentials to file and update environment
+# variables accordingly.
+#
+
+# check if the SSL file exists and if not assume it needs
+# the MESOS_SANDBOX prefix.
+write_cred_file()
+{
+    if [ -n "$DCOS_SERVICE_ACCOUNT_CREDENTIAL" ] && [ "$DCOS_SERVICE_ACCOUNT_CREDENTIAL" != file* ]; then
```
Comment (ssk2) [inline scripts/dcos-account.sh]: What does `file*` do? I can't easily Google it...

Comment (colin-msphere) [inline scripts/dcos-account.sh]: The bash equivalent of `startsWith`. This is checking whether the left side of the check starts with "file".

Comment (ssk2) [inline scripts/dcos-account.sh]: Aha!

File: scripts/dcos-account.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh 
+#
+# Write DC/OS credentials to file and update environment
+# variables accordingly.
+#
+
+# check if the SSL file exists and if not assume it needs
+# the MESOS_SANDBOX prefix.
```
Comment (adam-mesos) [inline scripts/dcos-account.sh]: Please update this comment. This is not `export-libssl.sh`

**Comment by colin-msphere** [comment]
@ssk2 dind version bumped.

**Pull request merged**